### DISCUSSION TITLE
[QUESTION] I am using AIOkafka, but i have some question.

### DISCUSSION THREAD

**Discussion by kings-ty** [aio-libs/aiokafka#934] [Q&A]
while I'm using AIOkafka but sometimes when I using 'consumer.commit()'
At that time, I found this error on the log
'commit cannot be completed since the group has already rebalanced'

so when i find some information.
then they said to me the AIOkafka have an autocommoit.
so I changed to passive-commit = False

I use this code 'consumer.commit()'.
I've written it the end of the process  after service part

service1.handler()
service2.handler()
service3.handler()

consumer.commit() 
like this

so but how can i overcome this horrible situation
i want to solve this problem of rebalanced error when i commit()

eventually, i changed poll size and ms time 

so please answer to me except for this option

thank you

**Comment by vmaurin** [comment]
Maybe my question could help you https://github.com/aio-libs/aiokafka/issues/929

A main difference between the java kafka client and aiokafka is that the group rebalancing is managed in a background coroutine, so the partition assignment could change while you are processing records and before you commit them.

For me it feels like in real production environment, with multiple consumer, the `commit()` method without argument should be avoided
